src/app/analyze/stack/stack-overview/stack-overview.component.ts

```diff
@@ -1,6 +1,6 @@
 import { Stack } from './../../../models/stack';
 import { Component, OnInit } from '@angular/core';
-import { Router }            from '@angular/router';
+import { Router } from '@angular/router';
```

Comment:
By convention, the first lines of the import are angular imports, then a space and external libraries, and then internal references.